src/hooks/useENSRegistrationCosts.ts

```diff
@@ -87,7 +86,6 @@ export default function useENSRegistrationCosts({
     () => registrationParameters?.changedRecords || {},
     [registrationParameters?.changedRecords]
   );
-  const [debouncedChangedRecords] = useDebounce(changedRecords, 500);
```

Comment:
i don't see any re renders with this change, if I recall correctly the rerenders were caused because we were using different `records` values for different places where we use this hook as https://github.com/rainbow-me/rainbow/pull/3239/files#diff-26b2b451f7f45a7c8014de4223972843b268e85fa809900fa1bffb73b2082573L128

now we consume the records directly from the state, so the calculation is happening once per record change